drop-in/src/main/java/com/adyen/checkout/dropin/ActionHandler.kt

```diff
@@ -83,10 +83,10 @@ class ActionHandler(
         activity: FragmentActivity,
         provider: ActionComponentProvider<out BaseActionComponent<out Configuration>, out Configuration>
     ) {
-        getActionComponentFor(activity, provider, dropInConfiguration).apply {
+        getActionComponentFor(activity, activity.application, provider, dropInConfiguration).apply {
```

Comment:
I guess this has nothing to do with StatusRepository refactor, so it's better to be done in a separate PR.

Reply:
Fair enough, will keep that in mind for next time. It does however solve a big issue with the `StatusRepository`. The component wasn't destroyed after the fragment was, causing the polling to go on as long as the `DropInActivity` is alive

Reply:
good catch, I didn't notice that.